src/ui-test/tests/deploy.kubernetes.run.test.ts

```diff
@@ -54,16 +56,22 @@ describe('Camel standalone file deployment using Camel JBang Kubernetes Run', fu
         await killTerminal();
         await editorView.closeAllEditors();
         // remove deployed integration from a local cluster
-        execSync(`jbang -Dcamel.jbang.version=${jbangVersion} camel@apache/camel kubernetes delete --name=demoroute`, { stdio: 'inherit', cwd: RESOURCES_PATH });
+        if (kubernetesRunParameters[0].includes('openshift')) {
+            // workaround: because of issues with 'camel kubernetes delete --name=<name>' in versions 4.8.1+ and OpenShift we need to cleanup using 'oc'
```

Comment:
Can you provide the Camel jira issue number in the comment please? it will help to know when this workaround can be removed later on.

Reply:
reported and added

Reply:
https://issues.apache.org/jira/browse/CAMEL-21702